README.md

```
@@ -1,212 +1,66 @@
 
-## Preview
+The logo of the project is a placeholder (docs/images/icon-nautobot-topology-views.png) - please replace it with your app icon, making sure it's at least 200x200px and has a transparent background!
 
-![Topology with light mode](doc/img/topology_light.png)
-![Topology with dark mode](doc/img/topology_dark.png)
+To avoid extra work and temporary links, make sure that publishing docs (or merging a PR) is done at the same time as setting up the docs site on RTD, then test everything.
+-->
 
-## Install
+<p align="center">
+  <img src="https://raw.githubusercontent.com/jdrew82/nautobot-plugin-topology-views/develop/docs/images/icon-nautobot-topology-views.png" class="logo" height="200px">
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)